suite-native/module-send/src/components/SendOutputsForm.tsx

```diff
@@ -0,0 +1,149 @@
+        })) ?? [],
+    isCoinControlEnabled: false,
+    hasCoinControlBeenOpened: false,
+    selectedUtxos: [],
+    feeLimit: '0',
+    feePerUnit: '0',
+    options: [],
+});
+
+export const SendOutputsForm = ({ accountKey }: SendFormProps) => {
+    const dispatch = useDispatch();
+    const debounce = useAsyncDebounce();
+    const { showToast } = useToast();
+    const navigation =
+        useNavigation<StackNavigationProps<SendStackParamList, SendStackRoutes.SendReview>>();
+
+    const account = useSelector((state: AccountsRootState) =>
+        selectAccountByKey(state, accountKey),
+    );
+    const networkFeeInfo = useSelector((state: FeesRootState) =>
+        selectNetworkFeeInfo(state, account?.symbol),
+    );
+    const sendFormDraft = useSelector((state: SendRootState) =>
+        selectSendFormDraftByAccountKey(state, accountKey),
+    );
+
+    const form = useForm<SendOutputsFormValues>({
+        validation: sendOutputsFormValidationSchema,
+        context: {
+            networkFeeInfo,
+            networkSymbol: account?.symbol,
+            availableAccountBalance: account?.availableBalance,
+        },
+        defaultValues: {
+            outputs: [
+                {
+                    address: 'bcrt1q7r9yvcdgcl6wmtta58yxf29a8kc96jkyyk8fsw',
+                    amount: sendFormDraft?.outputs[0]?.amount,
+                },
+            ],
+        },
+    });
+
+    const {
+        handleSubmit,
+        control,
+        getValues,
+        formState: { isValid, isSubmitting },
+    } = form;
+    const watchedFormValues = useWatch({ control });
+
+    const storeFormDraftIfValid = useCallback(async () => {
+        await dispatch(
+            sendFormActions.storeDraft({
+                accountKey,
+                formState: constructFormDraft(getValues()),
+            }),
+        );
+    }, [accountKey, dispatch, getValues]);
+
+    useEffect(() => {
+        if (watchedFormValues && isValid) debounce(storeFormDraftIfValid);
+    }, [isValid, storeFormDraftIfValid, watchedFormValues, debounce]);
+
+    const handleNavigateToReviewScreen = handleSubmit(async () => {
+        // Ignore another submit button clicks until is the submit not finished.
+        if (isSubmitting) return;
+
+        // TODO: navigate to SendFeeScreen instead, when ready
+        // issue: https://github.com/trezor/trezor-suite/issues/10871
+        navigation.navigate(SendStackRoutes.SendReview, { accountKey });
+        const response = await dispatch(
+            onDeviceTransactionReviewThunk({
+                accountKey,
+                formState: constructFormDraft(getValues()),
+            }),
+        );
+
+        if (isRejected(response)) {
+            // TODO: display error message based on the error code
+            showToast({ variant: 'error', message: 'Something went wrong', icon: 'closeCircle' });
+
+            navigation.navigate(SendStackRoutes.SendAccounts);
```

Comment:
The information from design is that it should behave this way. If user clicks this button, he goes back to the send account selection.

Reply:
Correct me if I'm wrong, but you're only navigating to this screen from SendAccounts - meaning navigation go back would do the same thing. 

Benefit of using `navigation.goBack` is that it would also work if we add new route from which you can navigate to send form. If user clicks back in that case, we would want him to be redirected back, not to send accounts.

Reply:
This will be revisited in the error handling follow-up PR. There will be no direct navigation in this callback, but the error state will be set. Then the module will be able to detect this error in a useEffect and handle it according to its context (in this case it will trigger the navigation action).

Issue: https://github.com/trezor/trezor-suite/issues/13125